src/main/java/org/radarcns/stream/phone/PhoneUsageStream.java

```diff
@@ -39,12 +37,16 @@
 
     private final PlayStoreLookup playStoreLookup;
 
-    public PhoneUsageStream(Collection<StreamDefinition> definitions, int numThread,
-            StreamMaster master, RadarPropertyHandler properties) {
-        super(definitions, numThread, master, properties, logger);
+    public PhoneUsageStream() {
         playStoreLookup = new PlayStoreLookup(CACHE_TIMEOUT, MAX_CACHE_SIZE);
     }
 
+    @Override
+    protected void initialize() {
+        defineSensorStream("android_phone_usage_event");
```

Comment:
This should be windowed as well

Reply:
See above, this is only meant to improve the output in HDFS, not for the dashboard.